src/uct/base/uct_md.c

```diff
@@ -552,3 +552,10 @@ int uct_md_is_mem_type_owned(uct_md_h md, void *addr, size_t length)
     return md->ops->is_mem_type_owned(md, addr, length);
 }
 
+int uct_md_is_hugetlb_used(uct_md_h md, uct_mem_h memh)
```

Comment:
why we need to pass md, can we use memh->alloc_md?

Reply:
how? this is a uct memh, not ucp.

Reply:
right